recipes/serial-monitor-rust/recipe.yaml

```diff
@@ -15,41 +15,29 @@ build:
   script:
     - if: osx
       then:
-        - clang --version
+        - export PATH="/usr/bin:$PATH"  # Ensure that system Clang is used, not a potentially conflicting Homebrew version
```

Comment:
This should use conda-forge compilers.

Reply:
Yeah, but somehow this is the reason why it fails with apple-sys. The conda compilers are CLANG 19, but apple-sys requires 15..

Reply:
The reason is the error here:

```
 /Library/Developer/CommandLineTools/usr/lib/clang/15.0.0/include/module.modulemap:10:8: error: redefinition of module '_Builtin_intrinsics'
 │ │   $BUILD_PREFIX/lib/clang/15.0.0/include/module.modulemap:10:8: note: previously defined here
 │ │   /Library/Developer/CommandLineTools/usr/lib/clang/15.0.0/include/module.modulemap:156:8: error: redefinition of module '_Builtin_stddef_max_align_t'
 │ │   $BUILD_PREFIX/lib/clang/15.0.0/include/module.modulemap:156:8: note: previously defined here
 │ │   /Library/Developer/CommandLineTools/usr/lib/clang/15.0.0/include/module.modulemap:160:8: error: redefinition of module 'opencl_c'
 │ │   $BUILD_PREFIX/lib/clang/15.0.0/include/module.modulemap:160:8: note: previously defined here
```